ansible/veos_vtb

```diff
@@ -102,7 +103,7 @@ all:
           ansible_hostv6: fec0::ffff:afa:2
           type: kvm
           hwsku: Force10-S6100
-          serial_port: 9000
+          serial_port: 9095
```

Comment:
Why use a different console port for `vlab-02`?

Reply:
It is because there is conflict between vlab-01 and vlab-02, both of them are using serial port 9000. So I changed it.